com.microsoft.mrtk.input/Visualizers/RiggedHandVisualizer/RiggedHand-Main.mat

```diff
@@ -2,14 +2,16 @@
 %TAG !u! tag:unity3d.com,2011:
 --- !u!21 &2100000
 Material:
-  serializedVersion: 6
+  serializedVersion: 8
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_Name: RiggedHand
-  m_Shader: {fileID: 4800000, guid: 7e5190cdaf0ecda4c8cd1c9d0c6e017b, type: 3}
-  m_ShaderKeywords: _ENABLE_
+  m_Name: RiggedHand-Main
+  m_Shader: {fileID: 4800000, guid: 618b0da8e4e6bb2459a1cc27acf0dd57, type: 3}
+  m_ValidKeywords: []
+  m_InvalidKeywords:
+  - _ENABLE_
```

Comment:
I'm not super familiar with the serialization of materials, but is the move of `_ENABLE_` from `m_ShaderKeywords` to `m_InvalidKeywords` expected or bad?

Reply:
Honestly no clue. @Cameron-Micka ?

Reply:
I'm not 100% since I think this is new to 2021. But my guess is it is a keyword that is serialized to the material but not being used by the shader. ie 618b0da8e4e6bb2459a1cc27acf0dd57 doesn't have an `_ENABLE_` keyword. 

Materials endlessly snowball properties in case you switch shaders your properties are preserved.

Reply:
Resolved with the re-serialization!